packages/mdc-list/README.md

```diff
@@ -266,15 +391,21 @@ Mixin | Description
 
 The MDCList JavaScript component implements the WAI-ARIA best practices for
 [Listbox](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox). This includes overriding the default tab behavior
-within the list component. You should not add `tabindex` to any of the `li` elements in a list.
+within the list component.
+
+The `tabindex` should be set to `0` for first list item element or selected list item element, remaining list item elements should not have `tabindex` set to `true`.
```

Comment:
is `true` supposed to be `0` here? Or just leave it at "should not have `tabindex` set"?